src/test/java/codeu/model/data/ConversationTest.java

```diff
@@ -34,5 +34,9 @@ public void testCreate() {
     Assert.assertEquals(owner, conversation.getOwnerId());
     Assert.assertEquals(title, conversation.getTitle());
     Assert.assertEquals(creation, conversation.getCreationTime());
+    
+    //Since Conversation and Message both implement Activity, we test the compareTo method
+    Message message = new Message(UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID(), "Test2", Instant.now().plusSeconds(2));
```

Comment:
Let's move this to a new test method, testCompareTo(), since this logic is not really in the create() method.